instrumentation/apache-camel-2.20/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/ActiveSpanManager.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel;
+/*
+ * Includes work from:
```

Comment:
I'm trying this pattern and it seems to work well to keep the copyrights together, what do you think?

https://github.com/open-telemetry/opentelemetry-java/blob/master/build.gradle#L331

Also if the original isn't shortened I don't know if we're allowed to - IANAL but my understanding is we need to preserve the original header without modification.

Reply:
Excellent idea! Reformatted to put "includes" header right after our license. Also, unwinded the original one per suggestion.